docs/source/api.rst

```diff
@@ -420,6 +396,52 @@ Transformers/encoders
      :members:
 
 
+Clustering
+~~~~~~~~~~
+
+.. autosummary::
+
+    vaex.ml.cluster.KMeans
+
+.. autoclass:: vaex.ml.cluster.KMeans
+     :members:
+
+
+Metrics
+~~~~~~~
+
+.. autosummary::
+
+     vaex.ml.metrics.DataFrameAccessorMetrics.accuracy_score
+     vaex.ml.metrics.DataFrameAccessorMetrics.precision_score
+     vaex.ml.metrics.DataFrameAccessorMetrics.recall_score
+     vaex.ml.metrics.DataFrameAccessorMetrics.f1_score
+     vaex.ml.metrics.DataFrameAccessorMetrics.confusion_matrix
+     vaex.ml.metrics.DataFrameAccessorMetrics.precision_recall_fscore
+     vaex.ml.metrics.DataFrameAccessorMetrics.matthews_correlation_coefficient
+     vaex.ml.metrics.DataFrameAccessorMetrics.classification_report
+     vaex.ml.metrics.DataFrameAccessorMetrics.mean_absolute_error
+     vaex.ml.metrics.DataFrameAccessorMetrics.mean_squared_error
+     vaex.ml.metrics.DataFrameAccessorMetrics.r2_score
```

Comment:
Why not?
```suggestion

.. autoclass:: vaex.ml.metrics.DataFrameAccessorMetrics
     :members:
```
Maybe this didn't work because the class didn't have a docstring? (see comment below)

Reply:
Indeed, it seems to be due to the lack of docstring in the dataframe accessor classed. Fix in the upcoming commit. Thanks!